libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -273,7 +285,8 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
 #endif // AIRSPEED_MAX_SENSORS
 
     // Note that 21, 22 and 23 are used above by the _OPTIONS, _WIND_MAX and _WIND_WARN parameters.  Do not use them!!
-    
+
+    // NOTE: Index 63 is used by AIRSPEED_TYPE: This is do its 0 index being treated as 63 Do not use it!
```

Comment:
```suggestion
    // NOTE: Index 63 is used by AIRSPEED_TYPE: This is due its 0 index being treated as 63 Do not use it!
```
... these comments aren't making much sense to it.  More words needed! :-)

Reply:
;) Done